app/src/androidTest/java/org/mozilla/focus/activity/ToggleBlockTest.java

```diff
@@ -73,10 +71,22 @@ public void tearDown() throws Exception {
     @Test
     public void SimpleToggleTest() throws UiObjectNotFoundException {
         // Load mozilla.org
-        onView(withId(R.id.url_edit))
-                .check(matches(isDisplayed()))
-                .check(matches(hasFocus()))
-                .perform(typeText("mozilla"), pressImeActionButton());
+        //
+        //onView(withId(R.id.url_edit))
+        //        .check(matches(isDisplayed()))
+        //        .check(matches(hasFocus()))
+        //        .perform(typeText("mozilla"), pressImeActionButton());
+        // Above types "mmozilla"
```

Comment:
We had this problem before and started to use replaceText() instead of typeText(). This seemed to work.